docs/manuals/source/DeveloperGuide/release.rst

```diff
@@ -0,0 +1,74 @@
+So the version you're going to release is probably the final version for that pre-release.
+
+Preparing the release notes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+For each new release there should be a section in the :ref:`releasenotes`.
+You can use the release notes of a previous release as a template.
+
+The release notes should be committed to the master branch and then cherry-picked to your release branch using ``git cherry-pick -x``.
+
+Update version-dependent files
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+There are a few version-dependent files in the Bareos sources that need to be changed.
+
+version.h
+^^^^^^^^^
+Starting with Bareos :sinceVersion:`19:remove version.h` the values for ``version.h`` are automatically determined by cmake based on git repository data.
+Changing ``version.h`` in such a new release is not neccessary anymore.
+
+If you are releasing an older version, the file is in ``core/src/include`` or ``src/include``.
+This file contains the Bareos version and some date and time values.
+
+You should set the following constants to new values:
+
+* VERSION to the version you're releasing
+* BDATE to the day you're releasing in strftime format "%d %B %Y"
+* LSMDATE to the date you're releasing in strftime format "%d%b%y" (no spaces!)
+* YEAR to the year you're releasing in strftime format "%Y"
+
+Also change the year in PROG_COPYRIGHT if applicable.
+
+version.map.in
+^^^^^^^^^^^^^^
+The file is in either ``core/src/cats/ddl`` or ``src/cats/ddl``.
+Whenever the database schema version changes this file must be updated.
+Usually the developer who did a schema change should have done this.
+However, please double-check and add or update the version mapping if needed.
+
+Committing your work
+~~~~~~~~~~~~~~~~~~~~
+The first line of your commit message should read "Release <your-version-number>", e.g. "Release 19.2.4".
+Tag your new commit with the release tag for your version ("Release/<your-version-number>"), e.g. ``git tag Release/19.2.4`` and push your changes to GitHub.
+
+.. important::
+   By pushing your release tag to GitHub the new version is released.
+
+Creating a work-in-progress tag
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+For automatic pre-release version generation for the next release a work-in-progress tag is required.
+Usually you will want to change the scheme of your pre-release versions when you release.
+To configure a new scheme, you need to create another tag.
+
+To allow several different pre-release branches based on your release, you should not tag the release commit you created earlier.
+Instead you should create an empty commit that immediately follows your release commit with ``git commit --allow-emtpy``.
+The first line of the commit message should read "Branch <your-next-version-number>", for example "Branch 19.2.5".
+That commit will then be tagged as ("WIP/<your-next-version-number>-pre"), for example ``git tag WIP/19.2.5-pre``.
+
```

Comment:
I'm not sure if I understood it correctly. You have to create an empty commit with the message "Branch <your-next-version-number>" and assign a WIP/... tag on it.
This is also done on master? However, you're not creating branches, just commits?
In my opinion, the commit message "Branch ..." is misleading, When working with git, I've another expectation.

How about "Beginning work for ..." or something else? Or just he same test as the tag (WIP/<your-next-version-number>-pre)?

Reply:
I used the term branch because these commits are usually the first commits that you would place on a new branch. However, as you can just add such a commit to an existing branch and there is no need to branch the term is misleading.

I like "Start development of". Maybe I can come up with something even better. I'll think about it again.

Reply:
"Start development of" sounds good.